burn-import/README.md

```diff
@@ -0,0 +1,65 @@
+# Burn Import
+
+`burn-import` is a crate designed to facilitate importing models trained in other machine learning
+frameworks into the Burn framework. This tool generates a Rust source file that aligns the source
+model with Burn's model and converts tensor data into a format compatible with Burn.
+
+Currently under development, `burn-import` supports importing ONNX models with a limited set of
+operators.
+
+## Supported ONNX Operators
+
+- Conv2d
+- Gemm (Linear layer)
+- Flatten
+- LogSoftmax
+
+## Usage
+
+### Importing ONNX models
+
+In `build.rs`, add the following:
+
+```rust
+use burn_import::onnx::ModelCodeGen;
+
+fn main() {
+    ModelCodeGen::new()
+        .input("src/model/mnist.onnx") // Path to the ONNX model
+        .out_dir("model/")            // Directory to output the generated Rust source file (under target/)
+        .run_from_script();
+}
+```
+
+Then, add the following to mod.rs under `src/model`:
+
+```rust
+pub mod mnist {
+    include!(concat!(env!("OUT_DIR"), "/model/mnist.rs"));
```

Comment:
We could provide an attributes that does everything during compile time.

```rust
#[burn(import = "src/model/mnist.onnx")]
struct ModelName<B>; // Going to be used as model name.
```

Reply:
That's a great idea. I can take this up in the next PRs. It should not be difficult to add this since the code is generated with TokenStream. I actually intended to use in macro, I was just trying to figure out a good syntax for it, which you provided just now.